src/main/java/com/codeless/api/automation/converter/NextTokenConverter.java

```diff
@@ -3,6 +3,7 @@
 import com.fasterxml.jackson.core.JsonProcessingException;
 import com.fasterxml.jackson.core.type.TypeReference;
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
```

Comment:
why do you need this import?

Reply:
to disable `SerializationFeature.FAIL_ON_EMPTY_BEANS`
        
        ```
return encode(objectMapper.disable(SerializationFeature.FAIL_ON_EMPTY_BEANS)
          .writeValueAsString(nextToken));
```